### PR TITLE
New component: sb-select

### PR DESCRIPTION
**Pull request by emanuelgsouza**

### CODE REVIEW

**Review by ademarCardoso — APPROVED** [review]
The component complies with the code standard and the design, for me it is ready to merge :rocket:

**Review by gustavomelki — COMMENTED** [review]
Some parameters of the methods remain to be commented. Besides that, excellent work!

**Comment by emanuelgsouza** [comment]
@gustavomelki , comments added! Could you check and merge?

File: src/components/Select/components/SelectListItem.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        : this.value === this.inputValue
+    },
+  },
+
+  methods: {
+    /**
+     * emits the 'emit-value' event
+     * @param {Event} event
+     */
+    handleClick(event) {
```
Comment (gustavomelki) [inline src/components/Select/components/SelectListItem.vue:73]: the event parameter isn't used here

**Review by gustavomelki — APPROVED** [review]
I did one more comment. After that, you can merge!

**Pull request merged**